src/templates/admin/elements/repository/version_detail.html

```diff
@@ -41,7 +46,16 @@ <h2>New Metadata</h2>
                     <p><strong>Published DOI{% if version.published_doi and version.published_doi != version.preprint.doi %} (changed){% endif %}</strong></p>
                     <p>{% if version.published_doi == version.preprint.doi %}Unchanged{% else %}{{ version.published_doi }}{% endif %}</p>
                     <p><strong>File</strong></p>
-                    <p>{% if version.file %}<a href="{{ version.file.download_url }}"><i class="fa fa-download"></i> Download</a>{% else %}No File{% endif %}</p>
+                    {% if version.file %}
+                    <p><a href="{{ version.file.download_url }}"><i class="fa fa-download"></i> Download</a></p>
```

Comment:
I think the icon here should be decorative not semantic.  see https://docs.fontawesome.com/web/dig-deeper/accessibility